src/lucky_cli/runner.cr

```diff
@@ -3,9 +3,18 @@ require "./text_helpers"
 
 class LuckyCli::Runner
   @@tasks = [] of LuckyCli::Task
+  @@exit_with_error_if_not_found = true
```

Comment:
You can use the `class_property` macro to set this up for you :)

Reply:
Ah ha! I was wondering if I was just missing that. Thanks!

Reply:
You're welcome. And for future reference, you can check out the "Macro summary" in https://crystal-lang.org/api/0.23.1/Object.html for a bunch of other goodies!